todo_app/templates/index.html

```diff
@@ -11,8 +11,16 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}  
+          <li>{{ item.title }} </li>  
+        {% endfor %} 
       </ul>
     </div>
   </div>
-{% endblock %}
\ No newline at end of file
+  <div>
+    <form action="http://127.0.0.1:5000/add">
+      <input type="submit" value="Add Item" />
+  </form>
```

Comment:
This should be an <a> element instead of a `<form>` when it's just a simple link.

And you should use `url_for('add')` like you did for the form on the other page.